src/features/components/molecules/deriv-products-hero/products-signup.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { hero_main_btn } from './deriv-products-hero.module.scss'
+import { Localize } from 'components/localization'
+import useAuthCheck from 'components/hooks/use-auth-check'
+import Button from 'features/components/atoms/button'
+import useHandleSignup from 'components/hooks/use-handle-signup'
+import { handleGetTrading } from 'components/custom/utils'
+
+const HeroMainButton = () => {
+    const [is_logged_in] = useAuthCheck()
+    const handleSignup = useHandleSignup()
+
+    if (is_logged_in) {
+        return (
+            <Button.Primary
+                className={hero_main_btn}
+                type={'button'}
+                onClick={handleGetTrading}
+                aria-label="Get trading"
```

Comment:
Button already has content. So, i think we don't need `aria-label` here

Reply:
sure bro thanks